google/cloud/aiplatform/experimental/vertex_model/base.py

```diff
@@ -379,10 +452,15 @@ class VertexModel(metaclass=abc.ABCMeta):
     dependencies = [
         "pandas>=1.3",
```

Comment:
Can we remove the pandas and torch dependencies?

Reply:
I wanted to keep these as the default, as we were keeping all other pytorch/pandas inclusion as a default as well. Removing it here would mean parsing all of the imports in the notebook to recreate these dependencies.